lib/util.js

```diff
@@ -291,6 +306,63 @@ exports.getVideoDescription = function(html) {
 
 
 /**
+ * Get video Owner from html.
+ *
+ * @param {String} body
+ * @return {Object}
+ */
+var authorRegexp = /<a href="\/channel\/([\w-]+)"[^>]+>(.+?(?=<\/a>))/
+var aliasRegExp = /<a href="\/user\/([^"]+)/
+exports.getAuthor = function(body) {
+  var ownerinfo = this.between(body, '<div id="watch7-user-header" class=" spf-link ">', '<div id="watch8-action-buttons" class="watch-action-buttons clearfix">');
```

Comment:
I didn't notice before, but you should refer to the `between` function by `exports.between` so that it's more explicit, and just in case someone ever calls `getAuthor` with a different context.